tests/phpunit/api/v3/ContactTest.php

```diff
@@ -1573,6 +1576,38 @@ public function testCreateIndividualWithNameEmail() {
     $this->callAPISuccess('contact', 'delete', $contact);
   }
 
+  /**
+   * Verify that attempt to create individual contact just email succeeds.
+   *
+   *
+   * @dataProvider versionThreeAndFour
+   * @param int $version
+   *
+   * @throws API_Exception
+   * @throws CRM_Core_Exception
+   */
+  public function testCreateIndividualWithJustEmail($version) {
+    $this->_apiversion = $version;
+    $params = [
+      'contact_type' => 'Individual',
+      'email' => 'garlic@example.com',
+    ];
+
+    $contact = $this->callAPISuccess('contact', 'create', $params);
```

Comment:
This is the actual function we are testing so it's the only one that varies by api version - in general prefer apiv4